IDEAS/Electrical/AC/ThreePhasesBalanced/Loads/Impedance.mo

```diff
@@ -141,6 +141,12 @@ When the connection type changes from
 is recomputed in such a way that the nominal power consumed by the impedance
 does not change.
 </p>
+<h4>References</h4>
+<p>
+[Kenelly 1899]: A.E. Kenelly, 
+\"Equivalence of triangles and stars in conducting networks\", 
+<i>Electrical World and Engineer</i>, vol. 34, pp. 413&ndash;414, 1899.
+</p>
```

Comment:
This is an IBPSA model, so it shouldn't be updated.
I would remove this reference overall (it's "common" knowledge in engineering).